draft-ietf-mls-protocol.md

```diff
@@ -1445,6 +1518,37 @@ the Group.
 It is RECOMMENDED for the application generating exported values
 to refresh those values after a group operation is processed.
 
+## Recovery Keys
+
+The main MLS key schedule provides a `recovery_secret` which can provide extra
+security in some cross-group operations.
+
+The application SHOULD specify an upper limit on the number of past
+epochs for which the `recovery_secret` may be stored.
+
+There are three ways in which a `recovery_secret` can be used: to re-initialize
+the group with different parameters, to re-add lost group members or to create a
+sub-group of an existing group, as detailed in {{pre-shared-keys}}.
+
+Recovery keys are distinguished from exporter keys in that they have specific
+use inside the MLS protocol, whereas the use of exporter secrets may be
+decided by an external application. They are thus derived separately to avoid
+key material reuse resulting from poor exporter key management external to
+the protocol.
+
+## State Authentication Keys
+
+The main MLS key schedule provides a per-epoch `authentication_secret`
+which MAY be used for authenticating the current group state. As with recovery
+keys, these are distinguished from exporter keys in that they have specific
+use affecting the MLS protocol.
+
+Members of a group MAY compare their `authentication_secrets` out-of-band to
+authenticate that they share the same view of the group.
+
+If one of the parties is being actively impersonated by an attacker, their
+`authentication_secret` will differ from that of the other group members.
```

Comment:
Try to compress this section into one paragraph

Reply:
I've shortened it a little bit and left it with two paragraphs. It should be in keeping with the style of the rest of the spec now.